example/test_problem/Hydro/Cosmic_Ray_Shocktube/generate_make.sh

```diff
@@ -0,0 +1,8 @@
+# This script should run in the same directory as configure.py
+
+PYTHON=python
+
+${PYTHON} configure.py --machine=eureka_intel --model=HYDRO --flu_scheme=MHM_RP --mhd=false --flux=HLLE \
```

Comment:
`--hdf5=true`

Reply:
I have updated the `Input__Parameter`. The script does not need HDF5 files for analysis.